src/test/resources/com/puppycrawl/tools/checkstyle/checks/trailingcomment/InputTrailingCommentPastFalsePositive.java

```diff
@@ -0,0 +1,129 @@
+package com.puppycrawl.tools.checkstyle.checks.trailingcomment;
+
+/**
+ * Config:
+ * default
+ */
+public class InputTrailingCommentPastFalsePositive {
+    public static boolean mybool = true; // violation
```

Comment:
is there other violations in this class? in case there are, it is not clear where are they without `// violation` comment

Reply:
No. There are no other violations. I added that because all input files must have `// violation` or `// ok`